run-tests.ps1

```diff
@@ -5,12 +5,22 @@ if (-Not $HasTestsDir -Or -Not $HasProjectDir) {
     exit 2
 }
 
+# Credit: https://stackoverflow.com/a/54935264
+function New-TemporaryDirectory {

```

Comment:
Codacy found an issue: [Function 'New-TemporaryDirectory' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.](https://app.codacy.com/manual/wwkimball/yamlpath/pullRequest?prid=6403599)